src/neuroconv/tools/data_transfers/_globus.py

```diff
@@ -22,7 +20,7 @@ def get_globus_dataset_content_sizes(
 
     Returns dictionary whose keys are file names and values are sizes in bytes.
     """
-    assert HAVE_GLOBUS, "You must install the globus CLI (pip install globus-cli)!"
+    assert is_package_installed(module_name="globus_cli"), "You must install the globus CLI (pip install globus-cli)!"
```

Comment:
Adjusted wording from `module` (as in `nwbinspector`) to `package` (as in `neuroconv`), which is also more accurate TBH

```suggestion
    assert is_package_installed(package_name="globus_cli"), "You must install the globus CLI (pip install globus-cli)!"
```